source/configure/config-proxy-apache2.rst

```diff
@@ -2,38 +2,95 @@
 
 .. _config-proxy-apache2:
 
-Configuring Apache2 as a proxy for Mattermost Server (Unofficial)
+Configuring Apache2 as a reverse proxy for Mattermost Server (Unofficial)
 -----------------------------------------------------------------
 
 .. important:: This unofficial guide is maintained by the Mattermost community and this deployment configuration is not yet officially supported by Mattermost, Inc. `Community testing, feedback and improvements are welcome and greatly appreciated <https://github.com/mattermost/docs/issues/1295>`__. You can `edit this page on GitHub <https://github.com/mattermost/docs/blob/master/source/configure/config-proxy-apache2.rst>`__.
 
-On a Debian-based operating system such as Ubuntu, Apache2 proxy configuration is done in the ``/etc/apache2/sites-available`` directory. Red Hat-based systems organize Apache configuration files `differently <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-web_servers>`__. If you're setting up Mattermost on a subdomain, you'll want to create a new configuration file along the lines of ``mysubdomain.mydomain.com.conf``.
+On a Debian-based operating system such as Ubuntu, Apache2 reverse proxy configuration is done in the ``/etc/apache2/sites-available`` directory. Red Hat-based systems organize Apache configuration files `differently <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-web_servers>`__. If you're setting up Mattermost on a subdomain, you'll want to create a new configuration file along the lines of ``mysubdomain.mydomain.com.conf`` and enable it afterwards (usually with ``a2ensite mysubdomain.mydomain.com``).
 
-**To configure Apache2 as a proxy**
+**To configure Apache2 (>= 2.4.47) as a reverse proxy**
 
 1. SSH into your server.
-2. Make sure the Apache modules ``mod_rewrite`` , ``mod_proxy``, ``mod_proxy_http``, and ``mod_proxy_wstunnel`` are installed and enabled. If not, follow the instructions from your Linux distribution to do so.
+2. Make sure the Apache modules ``mod_rewrite``, ``mod_proxy`` and ``mod_proxy_http`` are installed and enabled. If not, follow the instructions from your Linux distribution to do so (usually this is being done with ``a2enmod rewrite``, ``a2enmod proxy`` and ``a2enmod proxy_http``).
 3. Create the above mentioned configuration file. It is often helpful to start with a copy of ``000-default.conf`` or ``default-ssl.conf`` (on Ubuntu).
 4. Edit your configuration using the guide below:
 
-    1. If you're not setting up a subdomain, your ``ServerName`` will simply be set to ``mydomain.com``.
-    2. ``ServerAlias`` can been added too if you want to capture ``www.mydomain.com``.
+    1. If you're not setting up a subdomain, your ``ServerName`` will simply be set to ``mymattermost.tld``.
```

Comment:
@anx-ag - These sub-steps are currently displaying as 1., 2., 3., and should display as a., b., c. instead. You should be able to replace the numbers with alpha characters inline directly to address this.

Reply:
I did replace them with alphabet characters, the preview still shows roman numbers - let's see if it works in production then :)